src/components/FormComponents.tsx

```diff
@@ -21,6 +21,7 @@ export const FormWrapper = <FormikInput extends FormikValues>({
   <Formik<FormikInput>
     validateOnChange={false}
     validateOnBlur={false}
+    enableReinitialize={true}
```

Comment:
Myslim ze by to malo byt ok, ale Formik som zatial nepouzil tak neviem na 100% ci nieco niekde neexploduje.

Potrebujeme to v podstate preto lebo stav je prvykrat inicializovany s prazdnymi hodnotami a az na component mount sa pozrieme do localstorage a vytiahneme predosly stav ak nejaky je.

Ak by sme predosli stav vytiali rovno tam kde mame default hodnoty, server side a client side bude odlisny a react bude hadzat warning kvoli tomu.